utils.py

```diff
@@ -27,97 +28,73 @@ def hit_detection_and_move_bullets(
         if bt.move_bullet(dt):
```

Comment:
bt is een hele slechte variabele naam, ga je schamen

Reply:
![afbeelding](https://github.com/Finnyy012/TUPOLEVGAMING/assets/94832237/345aef0b-3657-4baf-9473-99bb5e2f2928)